ci/check-code-and-docs-gen.sh

```diff
@@ -16,8 +16,6 @@ if [ ! -f .gitignore ]; then
   echo "_output" > .gitignore
 fi
 
-make install-deps
```

Comment:
no because it is already called by the `generated-code` target https://github.com/solo-io/skv2/blob/ad7e6dee5b8f735967358f0004d57a71d9b39ba8/Makefile#L24